src/PSPackageProject.psm1

```diff
@@ -260,8 +260,10 @@ function ConvertPssaDiagnosticsToNUnit {
     $null = $sb.Append("Describe 'PSScriptAnalyzer Diagnostics' { `n")
     foreach ($d in $Diagnostic) {
         $ruleName = $d.RuleName
+        $scriptName = $d.ScriptName -replace "'", "``"
```

Comment:
I thought single quote escaping was `-replace "'","''"` (double the quotes)?

Reply:
I'm doing what the code is already doing on line 265 as it works.  I thought it was safer than trying to escape quotes.